### PR TITLE
feat: Adding generic calls, swpr setup and style changes

### PR DESCRIPTION
**Pull request by rossneilson**
Adding a swapr guild to the script and changing some styles to match the figma

### CODE REVIEW

File: hardhat.config.js [hunk 1]
```diff
@@ -35,6 +35,8 @@ module.exports = {
       'dxdao-contracts/contracts/erc20guild/implementations/SnapshotRepERC20Guild.sol',
       'dxdao-contracts/contracts/erc20guild/utils/GuildRegistry.sol',
       'dxdao-contracts/contracts/erc20guild/implementations/SnapshotERC20Guild.sol',
+      'dxdao-contracts/contracts/erc20guild/implementations/EnforcedBinaryGuild.sol',
+      // 'dxdao-contracts/contracts/erc20guild/implementations/EnforcedBinarySnapshotERC20Guild.sol',
```
Comment (MiltonTulli) [inline hardhat.config.js:39]: Should we remove this line comment?

File: src/components/Guilds/ToastNotifications/ToastNotificationContainer.tsx [hunk 2]
```diff
@@ -6,8 +6,9 @@ const ToastNotificationContainer = styled(ToastContainer)`
     top: 6em;
   }
   .Toastify__toast {
-    border: 1px solid ${({ theme }) => theme.colors.text};
+    /* border: 1px solid ${({ theme }) => theme.colors.text}; */
```
Comment (MiltonTulli) [inline src/components/Guilds/ToastNotifications/ToastNotificationContainer.tsx]: We can remove this I think

File: src/components/Guilds/Web3Modals/WalletModal.tsx [hunk 3]
```diff
@@ -43,7 +43,7 @@ const Divider = styled.hr`
 `;
 
 const GreyDivider = styled(Divider)`
-  border-top: 1px solid #c4c4c4;
+  border-top: 1px solid ${({ theme }) => theme.colors.muted};
```
Comment (MiltonTulli) [inline src/components/Guilds/Web3Modals/WalletModal.tsx]: Not 100% from your pr, but we have a Divider component in components/Guilds/common that's doing the same thing. Maybe a good spot for components refactor.

**Review by MiltonTulli — DISMISSED** [review]

File: hardhat.config.js (hunk 1 above)
Comment (rossneilson) [reply hardhat.config.js:39]: We'll need it later it just wasnt appearing in node modules for me right now

**Review by 0xvangrim — APPROVED** [review]
LGTM!

**Pull request merged**